<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-KJ2IMOL1" failures="0" tests="5" name="testcases.BookAFlightTest" time="0.009" errors="1" timestamp="2023-04-04T00:56:36 IST" skipped="4">
  <testcase classname="testcases.BookAFlightTest" name="bookingConfirmed" time="0.000">
    <skipped/>
  </testcase> <!-- bookingConfirmed -->
  <system-out/>
  <testcase classname="testcases.BookAFlightTest" name="searchForFlights" time="0.000">
    <skipped/>
  </testcase> <!-- searchForFlights -->
  <system-out/>
  <testcase classname="testcases.BookAFlightTest" name="chooseLowestPricedAirline" time="0.000">
    <skipped/>
  </testcase> <!-- chooseLowestPricedAirline -->
  <system-out/>
  <testcase classname="testcases.BookAFlightTest" name="purchaseFlightTicket" time="0.000">
    <skipped/>
  </testcase> <!-- purchaseFlightTicket -->
  <system-out/>
  <testcase classname="testcases.BookAFlightTest" name="Config" time="0.009">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at java.base/java.io.File.<init>(File.java:279)
at utilities.ConfigWriter.setPropertyValue(ConfigWriter.java:13)
at testcases.BaseTest.Config(BaseTest.java:30)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:317)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- Config -->
  <system-out/>
</testsuite> <!-- testcases.BookAFlightTest -->
